__fixtures__/v-next/outputv4/akash/deployment/v1beta1/deployment.ts

```diff
@@ -135,42 +135,42 @@ export interface MsgCloseDeploymentResponseSDKType {}
 /** DeploymentID stores owner and sequence number */
 export interface DeploymentID {
   owner: string;
-  dseq: Long;
+  dseq: bigint;
 }
 
 /** DeploymentID stores owner and sequence number */
 export interface DeploymentIDSDKType {
   owner: string;
-  dseq: Long;
+  dseq: bigint;
```

Comment:
should this be `BigInt` with caps included?

Reply:
Hi, I think bigint here is the way to use it as a basic type. For BigInt, it's more like a constructor for bigint. So for me I prefer bigint. 

And also, In fact at the beginning I used BigInt here, but my local compiler told me:

The left-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type.  ts(2362)
Operator '*' cannot be applied to types 'BigInt' and 'bigint'.  ts(2365)

So I changed it into bigint.